bignum.c

```diff
@@ -204,6 +204,20 @@ void bn_zero(bignum256 *a)
 	}
 }
 
+// sets a bignum to one.
+void bn_one(bignum256 *a)
```

Comment:
How is gnu99 related to inlines? There's plenty of methods which are already inline. I'm not sure bn_zero/bn_one should be inline.

First also we should decide whether bn_zero should have the loop unrolled.